src/components/TheNavigation.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="navigation-container">
+    <button
+      class="btn-back btn btn-nav"
+      :disabled="isBackBtnDisabled"
+      @click="goBack"
+    >
+      <font-awesome-icon
+        icon="fa-solid fa-angles-left"
+        size="xl"
+        class="icon-left"
+      />
+      Back
+    </button>
+    <button
+      class="btn-forward btn btn-nav"
+      :disabled="isForwardBtnDisabled"
+      @click="goForward"
+    >
+      Forward
+      <font-awesome-icon
+        icon="fa-solid fa-angles-right"
+        class="icon-right"
+        size="xl"
+      />
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    lastState: {
+      type: Array,
+    },
+
+    id: {
+      type: Number,
+      default: 0,
+    },
+  },
+  data() {
+    return {
+      index: 0,
+      isBackButtonClicked: false,
+      isForwardButtonClicked: false,
+      isBackBtnDisabled: true,
+      isForwardBtnDisabled: true,
+    };
+  },
+  methods: {
+    goBack() {
+      this.isBackButtonClicked = true;
+      this.isForwardButtonClicked = false;
+
+      this.lastState.forEach((state) => {
```

Comment:
če sem pikolovski, to kar se loopa ni `state` ampak nekaj drugega.. kaj shranjuješ v `lastState`?

Reply:
breed object je ubistvu `state`